autospec/pkg_integrity.py

```diff
@@ -223,6 +223,7 @@ def download_file(url, destination):
             curl.perform()
         except pycurl.error as e:
             print(e.args)
+            os.unlink(sign_file)
```

Comment:
Could we just not put the perform in a try catch? I don't think unlink inside the open block for the same file is very nice =/.

Reply:
Sure, I will fix...

Reply:
That's a good point. We still do this below when checking the HTTP code.